requirements.txt

```diff
@@ -4,6 +4,7 @@ PyYAML
 repoze.lru
 pbkdf2
 pycryptodome>=3.3.1
+future==0.16.0
```

Comment:
I don't see where this is used?  If it's not used, lets remove it as we've avoiding adding py2/3 compat library dependencies in the past due to overhead.